tests/FileTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * This file is part of the PHP Rebilly API package.
+ *
+ * (c) 2015 Rebilly SRL
+ *
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace Rebilly\Tests;
+
+use Rebilly\ApiKeyProvider;
+use Rebilly\Client;
+use Rebilly\Rest\File;
+use RuntimeException;
+
+/**
+ * Class FileTest.
+ *
+ * @group wip
+ */
+class FileTest extends TestCase
+{
+    public function testFile()
+    {
+        $client = new Client(['apiKey' => 'QWERTY']);
+
+        $client = new Client(
+            [
+                'apiKey' => ApiKeyProvider::env(),
+                'httpHandler' => function () use ($client) {
+                    $response = $client
+                        ->createResponse()
+                        ->withHeader('Content-Type', 'application/pdf')
+                    ;
```

Comment:
http://mnapoli.fr/approaching-coding-style-rationally/

Here, first section, about `Trailing commas` shows the reason to put commas, and I think it suitable for semicolon in method chain. Putting semicolon to new line, will keep diff cleanest after adding new line with new method call.